source/client/options_impl.cc

```diff
@@ -373,6 +401,16 @@ OptionsImpl::OptionsImpl(const nighthawk::client::CommandLineOptions& options) {
   trace_ = PROTOBUF_GET_WRAPPED_OR_DEFAULT(options, trace, trace_);
   open_loop_ = PROTOBUF_GET_WRAPPED_OR_DEFAULT(options, open_loop, open_loop_);
   tls_context_.MergeFrom(options.tls_context());
+
+  if (options.has_transport_socket()) {
+    envoy::api::v2::core::TransportSocket ts;
+    if (transport_socket_.has_value()) {
+      ts = transport_socket_.value();
```

Comment:
Sorry - one tiny optional nit, this line isn't covered in tests. If it's not a lot of effort, it would be nice to cover it,
because that keeps coverage up to 100% for this file.

Reply:
I realized this is the constructor, so there's never going to be an existing value in transport_socket_. Deleting the impossible branch brings us back to 100%.